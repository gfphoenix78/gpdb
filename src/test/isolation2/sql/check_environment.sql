select count(1) from gp_segment_configuration;
select dbid, content, role
  from gp_segment_configuration
  where status='d'
  order by dbid;
