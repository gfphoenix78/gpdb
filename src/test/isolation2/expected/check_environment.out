select count(1) from gp_segment_configuration;
 count 
-------
 8     
(1 row)
select dbid, content, role from gp_segment_configuration where status='d' order by dbid;
 dbid | content | role 
------+---------+------
(0 rows)
